src/FBProfile.ts

```diff
@@ -73,17 +70,18 @@ class FBProfile {
     this.linkURL = profileMap.linkURL;
     this.imageURL = profileMap.imageURL;
     this.userID = profileMap.userID;
-    this.email = Platform.OS === 'android' ? null : profileMap.email;
+    // TODO: [TS Migration]: Value should be undefined. Discuss before merging.
+    this.email = Platform.OS === 'android' ? null as unknown as undefined : profileMap.email;
```

Comment:
In this case we could just mark `email: string? | null` and we can avoid this unknown type

Reply:
This will break